src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/TierCalculationController.kt

```diff
@@ -6,21 +6,20 @@ import io.swagger.annotations.ApiResponse
 import io.swagger.annotations.ApiResponses
 import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
 import org.springframework.http.ResponseEntity
-import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.hmppstier.dto.TierDto
+import uk.gov.justice.digital.hmpps.hmppstier.service.SuccessUpdater
 import uk.gov.justice.digital.hmpps.hmppstier.service.TierCalculationService
 import uk.gov.justice.digital.hmpps.hmppstier.service.exception.EntityNotFoundException
 import java.util.UUID
 
 @Api
 @RestController
 @RequestMapping(produces = [APPLICATION_JSON_VALUE])
-@PreAuthorize("hasRole('ROLE_HMPPS_TIER')")
-class TierCalculationController(private val tierCalculationService: TierCalculationService) {
+class TierCalculationController(private val tierCalculationService: TierCalculationService, private val successUpdater: SuccessUpdater) {
```

Comment:
This class shouldn't even exist, should it? It's been moved to the delius-update project

Reply:
I'd like that functionality to be distinct from the tier calculation functionality, what should we wall the class? tierToDeliusUpdatesClient ?